libs/langchain/langchain/vectorstores/elasticsearch.py

```diff
@@ -786,7 +786,7 @@ def _search(
         def default_doc_builder(hit: Dict) -> Document:
             return Document(
                 page_content=hit["_source"].get(self.query_field, ""),
-                metadata=hit["_source"]["metadata"],
+                metadata=hit["_source"]["metadata"] if "metadata" in hit["_source"] else {},
```

Comment:
```suggestion
                metadata=hit["_source"].get("metadata", {}),
```

Could you also add tests for this please?